src/components/RandomJoke.js

```diff
@@ -1,8 +1,8 @@
 import { useEffect, useState, useContext } from "react";
 import { Row, CardTitle, CardText, Spinner } from "reactstrap";
-import { MainCard } from "./Card";
+import { MainCard } from "././styling/Card";
```

Comment:
One dot is enough 😄  `"./styling/Card"`

It is also in some other files, you can simplify it everywhere

Reply:
I've fixed this :)